cmd/thermal-recorder/.gitignore

```diff
@@ -0,0 +1 @@
+thermal-recorder
```

Comment:
You probably want a `/` in front of this in case there's a thermal-recorder directory somewhere